src/schema/survey.ts

```diff
@@ -75,34 +72,38 @@ export const survey = {
           "description": "Horizontal orientation (left or right)",
           "title": "Horizontal Position"
         },
-        "label": {
-          "description": "Text to display on button (e.g. 'Prev' or 'Next')",
-          "title": "Label",
+        "id": {
+          "description": "Unique identifier",
+          "title": "Id",
           "type": "string"
         },
         "link": {
           "description": "Link to tie to button click",
           "title": "Link",
           "type": "string"
         },
-        "mode": {
-          "$ref": "#/definitions/TButtonMode",
-          "description": "Render mode (link or button)",
-          "title": "Mode"
-        },
         "outline": {
           "description": "Show an outline",
           "title": "Outline",
           "type": "boolean"
         },
+        "title": {
+          "title": "Title",
+          "type": "string"
+        },
+        "type": {
+          "$ref": "#/definitions/TButtonMode",
```

Comment:
Curious why you're using the `$` prefix for `ref` here?

Reply:
This file is auto-generated. This is the JSON schema derived from the interfaces.